app/Http/Controllers/ControllerDash.php

```diff
@@ -130,6 +132,7 @@ public function dash() {
             $pyear = substr($now->year, -2);
         }
         $pwinner = Bronze::where('month', $pmonth)->where('year', $pyear)->first();
+        $pwinner_local = LocalHero::where('month', $month)->where('year', $year)->first();
```

Comment:
This should be `$pmonth` to get the previous month's winner and not the current

Reply:
You are correct, sir! Fixed.